### DISCUSSION TITLE
specific requirement for go 1.23

### DISCUSSION THREAD

**Discussion by taejoon** [gohugo-ananke/ananke#759] [2 - Q&A / Support]
I am using HUGO on gitlab.com with ananke theme. It worked fine until last week, but this week, it failed to be built with the following error. I suspect that "v0.0.0-20241022105359-0306b6a74db0" release has a specific requirement for go 1.23, which is not supported by gitlab.com yet. 

Can I resolve this with my setting? 

Thanks, 

----
go: downloading github.com/theNewDynamic/gohugo-theme-ananke v0.0.0-20241022105359-0306b6a74db0
go: github.com/theNewDynamic/gohugo-theme-ananke@v0.0.0-20241022105359-0306b6a74db0 requires go >= 1.23 (running go 1.22.8; GOTOOLCHAIN=local)
Error: failed to get ["-u" "github.com/theNewDynamic/gohugo-theme-ananke"]: failed to execute 'go [get -u github.com/theNewDynamic/gohugo-theme-ananke]': failed to execute binary "go" with args [get -u github.com/theNewDynamic/gohugo-theme-ananke]: go: downloading github.com/theNewDynamic/gohugo-theme-ananke v0.0.0-20241022105359-0306b6a74db0
go: github.com/theNewDynamic/gohugo-theme-ananke@v0.0.0-20241022105359-0306b6a74db0 requires go >= 1.23 (running go 1.22.8; GOTOOLCHAIN=local)
 *errors.errorString
Cleaning up project directory and file based variables
00:00
ERROR: Job failed: exit code 1

**Comment by davidsneighbour** [comment]
You can solve the issue by changing the go.mod of the theme back to 1.22

```go
module github.com/theNewDynamic/gohugo-theme-ananke

go 1.22
```

I'll add a patch in a bit.

**Comment by davidsneighbour** [comment]
https://github.com/theNewDynamic/gohugo-theme-ananke/commit/d544d76af70ba83a5bdce2b58a1fc2fc79d03882
